flow/display_list.cc

```diff
@@ -1416,10 +1466,18 @@ void DisplayListBuilder::drawPoints(SkCanvas::PointMode mode,
       return;
   }
   CopyV(data_ptr, pts, count);
+  // drawPoints treats every point or line (or segment of a polygon)
+  // as a completely separate operation meaning we cannot ensure
+  // distribution of group opacity without analyzing the mode and the
+  // bounds of every sub-primitive.
+  UpdateLayerOpacityCompatibility(false);
```

Comment:
I was under the impressoin that drawPoints bottoms out in a simple drawPath command in many cases, but I don't think I fully grasp them all.

At any rate it's probably not important to optimize this case, but it could cause a strange performance cliff...

Reply:
I discovered this via testing. Even in Polygon mode it has terrible overdraw issues:

https://fiddle.skia.org/c/228459001d2de8db117ce25ef5cedb0c